traffic_ops/testing/api/v3/servers_test.go

```diff
@@ -101,6 +97,70 @@ func GetTestServersQueryParameters(t *testing.T) {
 		t.Fatalf("Failed to get server by Delivery Service ID: %v", err)
 	}
 	params.Del("dsId")
+
+	resp, _, err := TOSession.GetServers(nil)
+	if err != nil {
+		t.Fatalf("Failed to get servers: %v", err)
+	}
+
+	if len(resp.Response) < 1 {
+		t.Fatalf("Failed to get at least one server")
+	}
+
+	s := resp.Response[0]
+
+	params.Add("type", s.Type)
+	if _, _, err := TOSession.GetServers(&params); err != nil {
```

Comment:
nit: I'm going to merge this anyways, but if you really want to level up this testing you could iterate over the servers and verify that the response only contains the properly filtered servers

Reply:
... I know...